docs/document/content/user-manual/shardingsphere-scaling/usage.cn.md

```diff
@@ -185,12 +207,25 @@ mysql> show scaling status 660152090995195904;
 | RUNNING                                           | 运行中                                                        |
 | EXECUTE_INVENTORY_TASK                            | 全量迁移中                                                     |
 | EXECUTE_INCREMENTAL_TASK                          | 增量迁移中                                                     |
-| ALMOST_FINISHED                                   | 基本完成                                                       |
-| FINISHED                                          | 已完成                                                         |
+| ALMOST_FINISHED                                   | 基本完成(偏内部使用的状态，不用特别在意这个状态。目前的设置逻辑：数据校验完成之后、切换配置之前)                                                       |
```

Comment:
It's not recommend to put spoken words here. If it's hard to understand, we could remove `ALMOST_FINISHED`, it's used internally.

Reply:
OK , I will remove the ALMOST_FINISHED